fenix/app/src/main/java/org/mozilla/fenix/shopping/ui/ProductReviewLoading.kt

```diff
@@ -0,0 +1,88 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.shopping.ui
+
+import androidx.compose.animation.core.LinearEasing
+import androidx.compose.animation.core.RepeatMode
+import androidx.compose.animation.core.StartOffset
+import androidx.compose.animation.core.animateFloat
+import androidx.compose.animation.core.infiniteRepeatable
+import androidx.compose.animation.core.rememberInfiniteTransition
+import androidx.compose.animation.core.tween
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.alpha
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.compose.annotation.LightDarkPreview
+import org.mozilla.fenix.theme.FirefoxTheme
+
+private const val ANIMATION_DURATION_MS = 1000
+private const val NUMBER_OF_BOXES = 5
+private const val INITIAL_ALPHA = 0.25f
+private const val TARGET_ALPHA = 1f
+private val boxHeights = listOf(80.dp, 80.dp, 192.dp, 40.dp, 192.dp)
```

Comment:
What are your thoughts on having these values as named consts aswell? E.g 
```
RELIABLE_BOX_HEIGHT = 80.dp, 
ADJUSTED_RATINGS_BOX_HEIGHT = 80.dp 
```
etc
Same comment for the animation list below

Reply:
Or having a data class such as BoxDesc/Info and having 1 list and a forEach 🙂

Reply:
Yeah, good point! I agree about the `BoxInfo` and having one list.

I think extracting/naming each height/offset is not necessary now with the `BoxInfo`, it's more clear what they mean. wdyt?

Reply:
Agreed, no need to extract the height/offsets 🙂